src/components/EditorCanvas/Table.jsx

```diff
@@ -219,10 +219,19 @@ export default function Table(props) {
                       <strong>Default :</strong>{" "}
                       {e.default === "" ? "Not set" : e.default}
                     </p>
+                    <p>
+                      <strong>Comment:</strong>{" "}
+                      {e.comment === "" ? (
+                        "Not comment" 
+                      ) : (
+                        <div>{e.comment}</div>
+                      )}
+                    </p>
                   </div>
                 }
                 position="right"
                 showArrow
+                style={{ width: "200px", wordBreak: "break-word" }}
```

Comment:
This breaks the UI. 

![image](https://github.com/drawdb-io/drawdb/assets/96800776/766d872e-27ef-43a8-8b98-5448dcbb442c)

Consider removing the width size and adding a 'max-w-[{width}px]' to the comment rather than the whole popover